examples/simple.py

```diff
@@ -5,18 +5,22 @@
 from quantuminspire.sdk.models.circuit import Circuit
 from quantuminspire.util.api.remote_backend import RemoteBackend
 
-with Circuit(platform_name="spin-2", program_name="prgm1") as c:
-    k = c.init_kernel("new_kernel", 2)
-    k.x(0)
-    k.hadamard(1)
-    k.measure(0)
-    k.measure(1)
+if __name__ == "__main__":
 
-print(c.content)
+    with Circuit(platform_name="spin-2", program_name="prgm1") as c:
+        k = c.init_kernel("new_kernel", 2)
+        k.x(0)
+        k.hadamard(1)
+        k.measure(0)
+        k.measure(1)
 
-backend = RemoteBackend()
+    print(c.content)
+    print(f"number of qubits: {c.max_number_of_qubits}")
```

Comment:
Why do you add this statement?

Reply:
Ah should have removed these again. This was for debugging some other weird behaviour:

```
# Generated by OpenQL 0.12.2 for program prgm1
version 1.0

qubits 10


.new_kernel
    x q[0]
    h q[1]
    measure q[0]
    measure q[1]

number of qubits: 2
Execution time in seconds: 0.3009359836578369
job_id: 1035
```
So even though we explicitly say that there are 2 qubits. OpenQL still compiles it to 10 qubits. I'll remove these debug statements again.